app-text/libstaroffice/libstaroffice-0.0.6.recipe

```diff
@@ -2,22 +2,33 @@ SUMMARY="StarOffice import library"
 DESCRIPTION="Import filter(based on librevenge) to import all StarOffice \
 documents."
 HOMEPAGE="https://github.com/fosnola/libstaroffice"
-COPYRIGHT="2015-2017 Laurent Alonso et al."
+COPYRIGHT="2015-2018 Laurent Alonso et al."
 LICENSE="MPL v2.0"
 REVISION="1"
-SOURCE_URI="http://github.com/fosnola/libstaroffice/releases/download/$portVersion/libstaroffice-$portVersion.tar.bz2"
-CHECKSUM_SHA256="86f396ae9a25a84cef4f45356132b283863a4657dcaff61eff4fd9f5bf3a2c08"
+SOURCE_URI="https://github.com/fosnola/libstaroffice/releases/download/$portVersion/libstaroffice-$portVersion.tar.bz2"
+CHECKSUM_SHA256="cc0f2d43d856e917c34711fbfde60114920f032713c4ce70bd535e07800f3c7d"
 
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
-SECONDARY_ARCHITECTURES="?x86"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
```

Comment:
Why do you create `commandSuffix`? It works exactly like `secondaryArchSuffix`, unless I'm missing something.

Reply:
That's part of the [commandBinDir](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#using-the-commandbindir-user-variable) trick to have the runtimes installed in `/system/bin/` instead of `/system/bin/x86/` when we build on x86 secondary arch.